<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_route</class>
 <widget class="QDialog" name="edit_route">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Редактирование маршртуа</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>381</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="loadingCityEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="unloadingCityEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="loadingDateEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="cargoWeightEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="cargoVolumeEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="cargoTypeEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="priceEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>400</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="acceptButton">
      <property name="text">
       <string>Ок</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="rejectedButton">
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>141</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Город погрузки</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Город разгрузки</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Дата погрузки</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Вес груза(т)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Объем груза(м3)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Тип груза</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Цена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>edit_route</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rejectedButton</sender>
   <signal>clicked()</signal>
   <receiver>edit_route</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
